src/ResourceManager/Sql/Commands.Sql/Database Backup/Cmdlet/GetAzureRmSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -0,0 +1,299 @@
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.Backup.Model;
+using Microsoft.Azure.Commands.Sql.Database.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Sql.Database_Backup.Cmdlet
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseLongTermRetentionBackup", DefaultParameterSetName = LocationNameSet, SupportsShouldProcess = true)]
+    public class GetAzureRmSqlDatabaseLongTermRetentionBackup : AzureSqlDatabaseLongTermRetentionBackupCmdletBase
+    {
+        /// <summary>
+        /// Parameter set name for backup name.
+        /// </summary>
+        private const string BackupNameSet = "BackupName";
+
+        /// <summary>
+        /// Parameter set name for database name.
+        /// </summary>
+        private const string DatabaseNameSet = "DatabaseName";
+
+        /// <summary>
+        /// Parameter set name for server name.
+        /// </summary>
+        private const string ServerNameSet = "ServerName";
+
+        /// <summary>
+        /// Parameter set name for location name.
+        /// </summary>
+        private const string LocationNameSet = "LocationName";
+
+        /// <summary>
+        /// Parameter set for using a Database Input Object when getting a single backup.
+        /// </summary>
+        private const string GetBackupByInputObjectSet = "GetBackupByInputObject";
+
+        /// <summary>
+        /// Parameter set for using a Database Input Object when getting multiple backups.
+        /// </summary>
+        private const string GetBackupsByInputObjectSet = "GetBackupsByInputObject";
+
+        /// <summary>
+        /// Parameter set for using a Database Resource ID when getting a single backup.
+        /// </summary>
+        private const string GetBackupByResourceIdSet = "GetBackupByResourceId";
+
+        /// <summary>
+        /// Parameter set for using a Database Resource ID when getting multiple backups.
+        /// </summary>
+        private const string GetBackupsByResourceIdSet = "GetBackupsByResourceId";
+
+        /// <summary>
+        /// The location the backups are in.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = LocationNameSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 0,
+            HelpMessage = "The location the backups are in.")]
```

Comment:
It might seem obvious to us, but consider that in geo restore scenario the server location and backup location are **not** the same, so it is important to make clear for LTRv2 where the backup location is supposed to be

Reply:
Updated help message.